src/components/DataTable/DataTable.styles.ts

```diff
@@ -32,6 +32,11 @@ export const GcxDataTableTbody = styled.tbody`
 
 export const GcxDataTableTr = styled.tr`
   min-width: 0;
+  border-bottom: 1px solid #ebebeb;
```

Comment:
Question; is there a theme prop we can use here?

Reply:
Okay i will ask Lou if there is a token we can use then change it on my next pr :)